src/Uno.Templates/content/unoapp/.template.config/template.json

```diff
@@ -406,6 +406,13 @@
       "type": "parameter",
       "datatype": "bool"
     },
+    "analyzers": {
+      "displayName": "Analyzers",
+      "description": "Enable C# analyzers",
```

Comment:
Let's give this a better description... what do I get with these analyzers? Technically even without this Analyzers are enabled in the project... it's a specific subset that we're enabling here...